build.gradle

```
@@ -1,29 +1,41 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.10"
+    ext.versions = [
+            'kotlin': '1.5.21'
+    ]
+    ext.deps = [
+            // Kotlin
+            'kotlinStdLib'        : "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}",
+
+            // Android
+            'androidxCoreKtx'     : 'androidx.core:core-ktx:1.6.0',
+            'androidxAppcompat'   : 'androidx.appcompat:appcompat:1.3.1',
```

```suggestion
            'androidxAppCompat'   : 'androidx.appcompat:appcompat:1.3.1',
```
Not sure if this should be camel cased like that but it looks right to me

---

I thought about this one a little. `Appcompat` looked a little strange to me as well, but the library's artifact ID is `appcompat` which is why I ended up going with that.